src/clj/game/cards/assets.clj

```diff
@@ -681,7 +697,9 @@
     :events {:corp-turn-begins ability}})
 
    "Melange Mining Corp."
-   {:abilities [{:cost [:click 3] :effect (effect (gain :credit 7)) :msg "gain 7 [Credits]"}]}
+   {:abilities [{:cost [:click 3] :effect (effect (do (gain :credit 7)
+                                                      (trigger-event state side :corp-spend-click (:cid card) 3)))
```

Comment:
Why is this necessary? Why don't other cards need this trigger? Also, did you test this... the `do` is making me nervous and is unnecessary. (It's implicit inside `effect` and `req`.)